src/BuildCheck.UnitTests/TestAssets/SampleAnalyzerIntegrationTest/.editorconfig

```diff
@@ -0,0 +1,9 @@
+[*.csproj]
```

Comment:
Could you please clarigy the reason of removing the root = true line from the editor config? 
It is not exatly a blocker in CI, however on local envs it could result some inconsistencies while developing.